docs/IntegrationWithExistingApps.md

```diff
@@ -720,6 +720,8 @@ public boolean onKeyUp(int keyCode, KeyEvent event) {
 
 That's it, your activity is ready to run some JavaScript code.
 
+> If your app is targeting the Android `api level 23` or greater, make sure you have, for the development build, the `overlay permission` enabled. You can check it with `Settings.canDrawOverlays(this);`. This is required because, if your app produces an error in the react native component, the error view is displayed above all the other windows. Due to the new permissions system, introduced in the api level 23, the user needs to approve it.
```

Comment:
Any modules using a permission e.g. geolocation will crash the app when target API level is 23.

> This one is blocking, without this, the app will not work.

I won't say this is a blocking issue since it's purely a documentation thing.

I think adding this might give the false assurance that this is the only issue with API level 23 and everything else will work. Instead we should either fix the issues with API level 23 or document all the things that will break if you target API level 23.